backend/src/main/java/org/cryptomator/hub/api/VaultResource.java

```diff
@@ -276,15 +281,15 @@ public Response legacyUnlock(@PathParam("vaultId") UUID vaultId, @PathParam("dev
 		if (accessTokenSeats > license.getSeats()) {
 			throw new PaymentRequiredException("Number of effective vault users exceeds available license seats");
 		}
-
+		var ipAddress = request.remoteAddress().hostAddress();
```

Comment:
Did you test this with a proxy? I'm not sure if `remoteAddress()` returns the correct value when a request is forwarded.

Reply:
![image](https://github.com/user-attachments/assets/3f19ce38-08ba-4a9b-b0b0-cf6a63754021)

Yes, you can test it with the https://github.com/cryptomator/hub/pkgs/container/hub/359786809?tag=test-amd64 but make sure you set `QUARKUS_HTTP_PROXY_PROXY_ADDRESS_FORWARDING: true`

Reply:
See also [43d7634](https://github.com/cryptomator/hub/pull/320/commits/43d7634390cdefd089575ebcd7a5cecfdb90782a)